src/toil/batchSystems/lsf.py

```diff
@@ -167,7 +169,7 @@ def prepareBsub(cpu, mem, jobID):
                     mem = old_div(float(mem), 1024**3)
                     mem_resource = parse_memory_resource(mem)
                     mem_limit = parse_memory_limit(mem)
-                bsubMem = ['-R', 'select[type==X86_64 && mem > {m}] '
+                bsubMem = ['-R',
```

Comment:
Is `select[mem > ` not compatible with lsf10?

Reply:
Not compatible with all architectures. Would break if machine type isn’t x86_64. I don’t think there is a reason to hard-code this as it will be picked up as a system default, for example our default is select[type==rhel7_enterprise] but will adopt the hard-coded value when it is in the code. Mem is whatever is specified by default mem (—defaultMem) on the command line, or whatever is set by the system default (ours is 2.5G) for a job with no -R specified.

Reply:
https://github.com/DataBiosphere/toil/issues/2725

Reply:
This created a conflict which I just resolved based on your fix in master. This is going through travis again.